src/weppy.ts

```diff
@@ -0,0 +1,469 @@
+/// <reference path="../weppy.d.ts"/>
+
+// these seem not to be defined in window
+interface Window {
+	XMLHttpRequest?: XMLHttpRequest;
+	XDomainRequest?: XMLHttpRequest;
+	JSON?: JSON;
+}
+
+module WeppyImpl {
+
+	var PROTOCOL_VERSION = 3,
```

Comment:
Our guidelines for consts: https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#constants
TLDR, no special casing for consts